File: softverskarealizacijafront/src/Models/KvarDto.js
export default class KvarDto{
    constructor(data){
        this.id=data.id;
        this.vremmePrijave= data.vremmePrijave;
        this.vremeOtklanjanja = data.vremeOtklanjanja;
        this.client=data.client;
        this.node=data.node;
        this.stanjeKvara=data.stanjeKvara;
    }

}

File: SoftverskaRealizacijaBackend/SoftverskaRealizacijaBackend/Dto/KvarDto.cs
using static SoftverskaRealizacijaBackend.Models.Enumerations;

namespace SoftverskaRealizacijaBackend.Dto
{
    public class KvarDto
    {
        public int Id { get; set; }
        public DateTime VremePrijave { get; set; }
        public DateTime VremeOtkanjanja { get; set; }
        public int Client { get; set; }
        public int Node { get; set; }
        public StanjeKvara StanjeKvara { get; set; }
    }
}
